ScrapPackedLibrary/ScrapPackedFile.cs

```diff
@@ -217,12 +220,8 @@ private void ExtractDirectory(string p_packedPath, string p_destinationPath)
             try
             {
                 foreach (var file in metaData.fileList)
-                {
                     if (file.FilePath.StartsWith(p_packedPath))
-                    {
-                        ExtractFile(file.FilePath, destinationPath + file.FilePath.Substring(p_packedPath.Length), fsPacked);
-                    }
-                }
+                       ExtractFile(file.FilePath, destinationPath + file.FilePath.Substring(p_packedPath.Length), fsPacked);
```

Comment:
I'm ok with removing the {} …
But the indentation is wrong (missing one space)
But maybe do not change lines where stuff is correct but just not your style as long as you do have to make logic changes anyways.